src/test/java/com/baeldung/web/controller/BookControllerFindAllSummaryTest.java

```diff
@@ -87,8 +87,8 @@ public class BookControllerFindAllSummaryTest {
 
 	@Before
 	public void setUp() {
-		book1 = new Book("123", "Book1", "Author1", "Publisher1", "Summary1");
```

Comment:
Constructors are called wrongly as they have fewer parameters but the test files they are generated with more